BREAKING.md

```diff
@@ -10,6 +10,12 @@ There are a few steps you can take to write a good change note and avoid needing
 - Provide guidance on how the change should be consumed if applicable, such as by specifying replacement APIs.
 - Consider providing code examples as part of guidance for non-trivial changes.
 
+## 0.53 Breaking changes
+- [readOnly and readOnlyPermissions removed from container-loader](#readOnly-and-readOnlyPermissions-removed-from-container-loader)
+
+### `readOnly` and `readOnlyPermissions` removed from `container-loader`
```

Comment:
i think it would be better to clarify that it's being removed from `Container` in `container-loader` rather than directly from `container-loader`